crates/ruff_linter/src/logging.rs

```diff
@@ -152,7 +153,10 @@ pub fn set_up_logging(level: LogLevel) -> Result<()> {
         })
         .level(level.level_filter())
         .level_for("globset", log::LevelFilter::Warn)
-        .chain(std::io::stderr())
+        .chain(fern::Output::writer(
+            Box::new(colors::auto(std::io::stderr())),
+            "\n",
```

Comment:
For context, what's the second argument here? A separator?

Reply:
Yeah, this is fern's default line separator, so this isn't actually changing anything. But you have to specify it explicitly if you want to create your own fern::Output with your own writer.